### ISSUE TITLE
API

### ISSUE THREAD

**Issue by danpaulsmith** [nblintao/ParaGraphL#1]
Hi, I love this implementation, but would like to use a different visualisation library. Is there a way to decouple your GPU implementation from Sigma.js, and use the layout algorithm with another library? Thanks

**Comment by nblintao** [comment]
Thanks for letting me know. There is no out-of-the-box way to decouple for now, but most of the implementation is in function `createProgram` so you can try to reuse that part. Which library are you using? Are you using `D3`?

**Comment by danpaulsmith** [comment]
No, I'm wanting to use vis.js by Almende. Thanks by the way, I'll try it out. A second question is - can I request position updates from the algorithm every 33ms for example to animate the layout? (Within your sigma implementation).
